app/static/main.css

```diff
@@ -1,227 +0,0 @@
-﻿.container {
```

Comment:
We'll need this probably for the react app....... or some similar styling, so....